modules/member_iam/variables.tf

```diff
@@ -28,3 +28,9 @@ variable "project_roles" {
   description = "List of IAM roles"
   type        = list(string)
 }
+
+variable "prefix" {
+  description = "Prefix member or group or serviceaccount"
+  type        = string
```

Comment:
We should provide a default (`serviceAccount`) here to keep it backwards-compatible.

Reply:
ok